### PR TITLE
Launch generate comment magician command

### PR DESCRIPTION
**Pull request by trodge**
Combines https://github.com/GoogleCloudPlatform/magic-modules/pull/9567 and https://github.com/GoogleCloudPlatform/magic-modules/pull/9605

**Release Note Template for Downstream PRs (will be copied)**

```release-note:none

```

### CODE REVIEW

File: .ci/magician/github/reviewer_assignment.go [hunk 1]
```diff
@@ -14,7 +14,7 @@ var (
 )
 
 // Returns a list of users to request review from, as well as a new primary reviewer if this is the first run.
-func ChooseReviewers(firstRequestedReviewer string, previouslyInvolvedReviewers []string) (reviewersToRequest []string, newPrimaryReviewer string) {
+func ChooseCoreReviewers(firstRequestedReviewer string, previouslyInvolvedReviewers []string) (reviewersToRequest []string, newPrimaryReviewer string) {
```
Comment (melinath) [inline .ci/magician/github/reviewer_assignment.go:17]: Whoops, I did not intend for this change to sneak in - but it also probably doesn't hurt anything.

**Review by melinath — COMMENTED** [review]
this includes a lot of changes I made so I don't really want to say anything other than that it does include them :-)

File: .ci/magician/cloudbuild/build_trigger.go [hunk 2]
```diff
@@ -1,3 +1,18 @@
+/*
+* Copyright 2023 Google LLC. All Rights Reserved.
```
Comment (melinath) [inline .ci/magician/cloudbuild/build_trigger.go:2]: Generally it's nice to split changes like this into separate PRs to make review easier.

**Review by melinath — APPROVED** [review]
The changes look pretty straightforward. Could you double-check a couple of edge cases before merging:

- [ ] no diffs
- [ ] breaking change override label works

It looks like the test PR hasn't been used since the rebase; I think everything should be fine but would be great to double-check.
